### ISSUE TITLE
Move ignored columns to the registry?

### ISSUE THREAD

**Issue by gforcada** [plone/plone.app.content#121]
On https://github.com/plone/plone.app.content/blob/master/plone/app/content/browser/contents/__init__.py in ``FolderContentsView`` class there is a ``ignored_columns`` method with a static list of catalog columns that are to be hidden from the ``folder_contents`` view.

If someone, say me :-), wants to have any of those columns available the only way to do so is either fork p.a.content or override that method.

Wouldn't make sense to move this configuration, and probably others, in a plone.app.registry key so it would be a matter of changing some settings?

Is there any strong opinion against it?

**Comment by thet** [comment]
@gforcada I agree, ``ignored_columns`` and ``ignored_indexes`` (introduced in https://github.com/plone/plone.app.content/pull/122 ) would be perfect for having them in the registry. This would include having an upgrade step in ``plone.app.upgrade``.
plone.app.contents master is currently included in Plone 5.0 and 5.1. I'm not sure, if we should introduce a new registry setting for Plone 5.0. Maybe plone.app.content should be branched for 5.0 at this point?

**Comment by spereverde** [comment]
We have the same feature request.
Say we would like to add this, preferably for plone 6, on which branch would we best start a PR?

**Comment by davisagli** [comment]
@spereverde For Plone 6, currently use the master branch of each package. (You can always check in https://github.com/plone/buildout.coredev/blob/6.0/sources.cfg to see which branch of each package is officially part of Plone 6.)

It can probably go either in this package or in CMFPlone, whichever is easier. Well, it probably should go in CMFPlone, since it could be used by volto as well, and this package is part of the classic UI.

At this point it probably should not be merged until after the final release of Plone 6 (which I think is quite soon), since it is a new feature.

**Comment by mauritsvanrees** [comment]
Maybe `plone.base` would be better, in the interface for Site Settings? Just a suggestion.

Indeed not _right now_ please. ;-)

**Comment by spereverde** [comment]
If ok for everyone, we'll make a PR for ignored_columns and ignored_indexes here : https://github.com/plone/plone.base/blob/main/src/plone/base/interfaces/controlpanel.py - class ISiteSchema
afaik there is no specific control panel for folder_contents

**Comment by mauritsvanrees** [comment]
Seems a fine place at first glance.
